.github/workflows/review-bot.yml

```diff
@@ -16,6 +16,7 @@ permissions:
 jobs:
   review-approvals:
     runs-on: ubuntu-latest
+    environment: master
```

Comment:
In afraid this won’t work until we merge to master, as the event triggering this is `pull_request_target` and that uses the workflow file available in the master branch.

Reply:
yep, indeed! That's why I waited with this pr until I had a screenshot of the config, I don't like blindly merging untested changes.